perses/tests/test_repex.py

```diff
@@ -182,11 +190,249 @@ def test_RESTCapableHybridTopologyFactory_repex_charge_mutation():
                 f_ij, df_ij = analyzer.get_free_energy()
                 data[f"{wt_name}-{mutant_name}"] = {'free_energy': f_ij[0, -1], 'error': df_ij[0, -1]}
 
-        DDG = data['arg-ala']['free_energy'] + data['ala-arg']['free_energy'] \
-              - (data['lys-ala']['free_energy'] + data['ala-lys']['free_energy'])
-        dDDG = np.sqrt(data['arg-ala']['error'] ** 2
-                       + data['ala-arg']['error'] ** 2
-                       + data['lys-ala']['error'] ** 2
-                       + data['ala-lys']['error'] ** 2)
+        forward_fe = data['arg-ala']['free_energy'] + data['ala-lys']['free_energy']
+        forward_fe_error = np.sqrt(data['arg-ala']['error']**2 + data['ala-lys']['error']**2)
+        reverse_fe = data['lys-ala']['free_energy'] + data['ala-arg']['free_energy']
+        reverse_fe_error = np.sqrt(data['lys-ala']['error']**2 + data['ala-arg']['error']**2)
+
```

Comment:
I don't think these free energies and errors are broken down properly. I think something like this would be better:
```
arg_fe = data['arg-ala']['free_energy'] + data['ala-arg']['free_energy']
lys_fe = data['lys-ala']['free_energy'] + data['ala-lys']['free_energy']
arg_lys_diff = abs(arg_fe - lys_fe)
arg_lys_diff_error = np.sqrt(data['arg-ala']['error'] ** 2
                       + data['ala-arg']['error'] ** 2
                       + data['lys-ala']['error'] ** 2
                       + data['ala-lys']['error'] ** 2)
```
Note that `arg_lys_diff = abs(arg_fe - lys_fe)` should in fact be a difference, because we expect the ARG->ALA->ARG leg to have an equivalent free energy to that of LYS->ALA->LYS.

Reply:
Should be solved by changes in https://github.com/choderalab/perses/pull/1065/commits/629b20e4de1ab9fc6fdab4af6bba830e6d4e4aed